app/helpers/window.js

```diff
@@ -15,17 +15,41 @@ const autoResize = (dynamicWindow) => {
     }
   }
 
-  dynamicWindow.webContents.on('did-finish-load', () => {
-    const updateHeight = () => {
-      if (!dynamicWindow) { return }
-      dynamicWindow.webContents.executeJavaScript('document.body.children[0].offsetHeight', (mainContentHeight) => {
-        resize(mainContentHeight)
-      })
+  const updateHeight = () => {
+    if (!dynamicWindow) { return }
+    dynamicWindow.webContents.executeJavaScript('document.body.children[0].offsetHeight', (mainContentHeight) => {
+      resize(mainContentHeight)
+    })
+  }
+
+  const clearUpdateHeightInterval = () => {
+    if (dynamicWindow.updateHightIntervalId) {
+      clearInterval(dynamicWindow.updateHightIntervalId)
+      dynamicWindow.updateHightIntervalId = null
+    }
+  }
+
+  // register updateHeight interval only when dynamicWindow is visible
+  // remove the interval as soon as the dynamicWindow is not visible
+  const registerCallbacks = () => {
+    clearUpdateHeightInterval()
+    if (dynamicWindow.isVisible()) {
+      dynamicWindow.updateHightIntervalId = setInterval(updateHeight, 125)
```

Comment:
Instead of putting this on `dynamicWindow` can we just put it in the scope of this `autoResize` ? Similar to `currentHeight`.